franka.sh

```diff
@@ -53,6 +53,9 @@ echo "Starting Franka driver with hostname: $(hostname), Franka name: ${robot_na
 # we try to kill any existing instances, but if there are multiple containers
 # running on the same machine, or have an instance running on another
 # networked machine, we don't want to duplicate
+if [[ ":$PATH:" != *":/src/deploy/scripts:"* ]]; then
+    export PATH=$PATH:/src/deploy/scripts
+fi
```

Comment:
i don't think this should be added here. the path is already set up in our containers. if it's not there, that's some other bug.